docs/admin/install.rst

```diff
@@ -0,0 +1,265 @@
+
+  .. code-block:: sh
+
+    qvm-run --pass-io vault "gpg --homedir /run/media/user/TailsData/gnupg --export-secret-keys --armor <SVSFingerprint>" > /tmp/sd-journalist.sec
+
+  where ``<SVSFingerprint>`` is the submission key fingerprint, typed as a single unit without whitespace. This will copy the submission key in ASCII format to a temporary file in dom0, ``/tmp/sd-journalist.sec``. Verify the that the file starts with ``-----BEGIN PGP PRIVATE KEY BLOCK-----`` using the command:
+
+  .. code-block:: sh
+
+    head -n 1 /tmp/sd-journalist.sec
+
+- in the ``vault`` file manager, select **+ Other Locations** and eject the TailsData volume, then disconnect the SVS USB. 
+
+Copy Journalist Interface details 
+---------------------------------
+
+SecureDrop Workstation connects to your SecureDrop instance's API via the Journalist Interface. In order to do so, it will need the Journalist Interface address and authorization info. This can be copied from the Admin Workstation via the following steps:
+
+- Connect the Admin Workstation USB to a USB port on the Qubes computer, then use the devices widget in the upper right panel to attach it to the ``vault`` VM. There will be 3 listings for the USB in the widget: one for the base USB, one for the Tails partition on the USB, labeled ``Tails``, and a 3rd unlabeled listing, for the persistent volume. Choose the third listing.
+
+- In the the ``vault`` file manager, select **+ Other Locations**., then click the persistent volume's listing in the right panel. It will be named ```N GB encrypted``, where N is the size of the persistent volume. Enter the Admin Workstation persistent volume passphrase to unlock and mount it. 
+
+- Copy the Journalist Interface configuration file to `dom0`. If your SecureDrop instance uses v3 onion services, use the following command:
+
+  .. code-block:: sh
+
+    qvm-run --pass-io vault "cat /run/media/user/TailsData/Persistent/securedrop/install_files/ansible-base/app-journalist.auth_private" > /tmp/journalist.txt
+
+  If your instance uses legacy v2 onion services, use the following command instead:
+
+  .. code-block:: sh
+
+    qvm-run --pass-io vault "cat /run/media/user/TailsData/Persistent/securedrop/install_files/ansible-base/app-journalist-aths" > /tmp/journalist.txt
+
+- Verify that the `/tmp/journalist.txt` file on ``dom0`` contains valid configuration information using the command ``cat /tmp/journalist.txt`` in the administration terminal.
+
+- In the ``vault`` file manager, select **+ Other Locations** and eject the TailsData volume, then disconnect the Admin Workstation USB. Shut down the ``vault`` VM using the Qube widget in the upper right panel.
+
+
+Download and install SecureDrop Workstation
+-------------------------------------------
+
+With the key and configuration available in ``dom0``, you're ready to set up SecureDrop Workstation:
+
+- First, re-enable the network connection using the network manager widget.
```

Comment:
For once, copy and paste will actually work; how about we instruct them to visit this documentation in a browser at this point, so they can copy/paste the commands, don't have to type full fingerprint by hand, etc.?

Reply:
IMO still worth addressing explicitly, there's a lot of manual tippety-typing in this section otherwise.

Reply:
I'm against it, just because it's a bit of a context switch and I want to keep the amount of things happening on the workstation itself to a minimum.

Reply:
Can you clarify what process you are suggesting? Are you recommending they type in all the commands and file contents? That seems laborious and error-prone, especially since it includes a fingerprint (twice). IMO this is not a minor issue -- copy/paste in Qubes is very much nonstandard, and we're extremely careful about when we suggest using a browser or network-enabled VMs, so conservative users may assume it is _not_ safe to use a browser, which is not the case.

As a simple tip I don't think it adds a significant context switch, e.g.: "If you do not want to manually type the following commands, you can safely open a web browser in the `work` VM and copy and paste directly from this documentation.`

Reply:
Ok, but if they choose to do so, then you have to explain in which contexts they can use copy/paste. Every choice and every optional window is extra mental overhead, which increases the chance of people getting lost in the install process. Keeping the instructions as linear as possible (at the cost of having them type some commands) reduces that overhead.

Reply:
In general, I don't think we have the luxury to avoid talking about copy and paste. It's an essential operation in any operating system, and we will need to add a section to the overall docs explaining what does and doesn't work, and why. We will update this as we make adjustments to that policy, which I'm quite certain we will need to do given how conservative it currently is.

In this context, we're asking the user to create configuration files from scratch, import fingerprints, etc. So context-switch or not, I do think we need to give them a recommended option for avoiding having to manually type all those instructions.

Reply:
Fair enough, I'll add in a note here.